### DISCUSSION TITLE
Is there any way to restrict storing credentials in webview2 cache?

### DISCUSSION THREAD

**Discussion by apps1232** [MicrosoftEdge/WebView2Feedback#3940] [Q&A]
I have a webview2 based application. When I signed in to this, the user credentials got saved into the webview2 cache. Already we are having a cache clearing mechanism, but which is during the startup of our application. So, when there is situations in between startup and stop, the user credentials are saved in cache. And this saved credentials causes single sign on in some special cases, that I want to prevent.

Is there any way to avoid saving this user credentials in webview2 cache?

**Comment by champnic** [comment]
Hey @apps1232 - Can you see if the IsPasswordAutosaveEnabled setting works for your scenario?
https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2settings.ispasswordautosaveenabled?view=webview2-dotnet-1.0.1108.44

**Comment by apps1232** [comment]
> Hey @apps1232 - Can you see if the IsPasswordAutosaveEnabled setting works for your scenario? https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2settings.ispasswordautosaveenabled?view=webview2-dotnet-1.0.1108.44

Hi, Thank you for your reply. I have tried IsPasswordAutosaveEnabled setting..But my issue did not get solved.
But i tried another method, ie  CallDevToolsProtocolMethod("Network.clearBrowserCookies", "{}"); will clear the cookies.
Also, we can use CoreWebView2.CookieManager.DeleteAllCookies(); to delete the cookies. These two methods are working.
